authentication/passwordless/passwordless.go

```diff
@@ -5,52 +5,71 @@ import "github.com/auth0/go-auth0/authentication/oauth"
 // SendEmailRequest defines the request body for starting a passwordless flow via email.
 type SendEmailRequest struct {
 	oauth.ClientAuthentication
-	Connection string                 `json:"connection,omitempty"`
-	Email      string                 `json:"email,omitempty"`
-	Send       string                 `json:"send,omitempty"`
+	Connection string `json:"connection,omitempty"`
+	// The users's email address.
+	Email string `json:"email,omitempty"`
+	// Use `link` to send a link or `code` to send a verification code. If omitted, a `link` will be sent.
+	Send string `json:"send,omitempty"`
+	// Append or override the link parameters (like scope, redirect_uri, protocol, response_type), when you send a link.
 	AuthParams map[string]interface{} `json:"authParams,omitempty"`
 }
 
 // SendEmailResponse defines the response from the `SendEmail` request.
 type SendEmailResponse struct {
-	ID            string `json:"_id,omitempty"`
-	Email         string `json:"email,omitempty"`
-	EmailVerified bool   `json:"email_verified,omitempty"`
+	// The identifier of the request.
+	ID string `json:"_id,omitempty"`
+	// The user's email address.
+	Email string `json:"email,omitempty"`
+	// Whether the user's email address is verified.
+	EmailVerified bool `json:"email_verified,omitempty"`
 }
 
 // LoginWithEmailRequest defines the request body for exchanging a code requested by `SendEmail` for a token.
 type LoginWithEmailRequest struct {
 	oauth.ClientAuthentication
-	Audience  string `json:"audience,omitempty"`
-	Code      string `json:"otp,omitempty"`
+	// API Identifier of the API for which you want to get an Access Token.
```

Comment:
```suggestion
	// API Identifier of the API for which you want to get an access token.
```
According to the Docs team, we should not title-case standard industry terms like access token.

Reply:
Gah that always catches me out sorry!